docs/en/settings/color-settings.md

```diff
@@ -116,8 +112,30 @@ When planning your pages, make sure there is an even distribution and well-balan
   </div>
 </div>
 
-<hr />
+You can define a brand color (`$color-brand`) that can be used for call-to-action buttons and other highlights across the framework. The default Vanilla brand color is `$color-dark`.
 
-### Design
+<img class="p-image--bordered" src="https://assets.ubuntu.com/v1/7446a44a-basics-brand-color.png" alt="brand-color">
+
+### Accessibility
 
-* [Color design](https://github.com/ubuntudesign/vanilla-design/tree/master/Color)
+It’s important for us to meet all web accessibility standards. Vanilla encourages meeting the minimum contrast ratios specified by WCAG 2.1 Level AA for text, icons and background colors.
+
+<div class="row">
+   <div class="col-6">
+   <div class="p-notification--positive">
+   <p class="p-notification__response"><span class="p-notification__status">Do:</span>Minimal contrast ratio: 4.5</p>
+   </div>
+   <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/9e60d838-basics-text-color-do.png" alt="text-color-do">
+   <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/a55aa80e-basics-button-color-do.png" alt="button-color-do">
+   </div>
+  <div class="col-6">
+  <div class="p-notification--negative">
+  <p class="p-notification__response"><span class="p-notification__status">Don't:</span>Avoid using dark text that isn’t legible on dark backgrounds</p>
+  </div>
+  <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/9f98d99d-basics-text-color-don%27t.png" alt="text-color-don't">
+  <img class="p-image--bordered" src="https://assets.ubuntu.com/v1/4b8dc06e-basics-button-color-don%27t.png" alt="button-color-don't">
+  </div>
+</div>
+
+### Design
+Click [here](https://github.com/ubuntudesign/vanilla-design/tree/master/Color) to view Color design spec, which includes build MD file and PNG image.
```

Comment:
We shouldn't use "Click here", links should always be semantic descriptions of where they go:
```suggestion
For more information [view the color design spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Color), which includes the specification in markdown format and a PNG image.
```